### ISSUE TITLE
libzmq.dll vs zmq.dll

### ISSUE THREAD

**Issue by stisa** [nim-lang/nim-zmq#17]
Any particular reason why the dll on windows is called `zmq.dll` instead of `libzmq.dll` ?  
Looking at packages in https://zeromq.org/download/ , they all provide `libzmq.dll`.

**Comment by Araq** [comment]
When I wrote the code it was ``zmq.dll``. (Btw ``lib`` prefixes are pure visual noise.)

**Comment by stisa** [comment]
> (Btw lib prefixes are pure visual noise.)

I agree, it's just that I updated jupyter which I guess updated my zmq lib because suddenly my jupyter kernel couldn't load `zmq.dll` anymore.
